src/components/JsonLink/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRobot } from '@fortawesome/free-solid-svg-icons';
+import styles from './index.module.scss';
+
+interface Props {
+  fileName: string;
+  version: string;
+}
+
+const baseURL = `https://nodejs.org/docs/latest-`;
+
+const JsonLink = ({ fileName, version }: Props): JSX.Element | null => {
+  const href = `${baseURL}${version}.x/api/${fileName}.json`;
```

Comment:
This could be inline on the `<a href` no? So that you don't need the `return ()` statement 👀